man/dnstable_convert.1.txt

```diff
@@ -62,3 +62,12 @@ be useful.
 
 ^-p^::
     Preserve empty DNS/DNSSEC MTBL files.
+
+^-r^::
+    Emit RDATA and RDATA_RNAME_REV dnstable entries for SOA rname field.
+
+^-s NAME^::
+    NMSG source information to include in output if input is stdin.
```

Comment:
Note that `-s` can be used with file inputs, in which case it overrides the filename. Also, "NMSG source" may be confused with the NMSG payload `source` field.

Suggest:
```
Source metadata to include in output. Required if input is stdin, otherwise defaults to input file name.
```

Reply:
Done.